Library/Formula/jboss-forge.rb

```diff
@@ -1,12 +1,17 @@
 class JbossForge < Formula
   desc "Tools to help set up and configure a project"
   homepage "http://forge.jboss.org/"
-  url "https://downloads.jboss.org/forge/releases/2.20.0.Final/forge-distribution-2.20.0.Final-offline.zip"
-  version "2.20.0.Final"
-  sha256 "ee4f9b2fff8ac72714cfd0c852735fcdd04e1b4462de8a6ba0807cbe3df4503f"
+  url "https://downloads.jboss.org/forge/releases/2.20.1.Final/forge-distribution-2.20.1.Final-offline.zip"
+  version "2.20.1.Final"
+  sha256 "dbe51b8c9cd0cf1b0a9adadcd8161c442a4ab586ca63f3a3f6d3369a2831afae"
 
   bottle :unneeded
-
+  
+  devel do
+    url "https://downloads.jboss.org/forge/releases/3.0.0.Alpha1/forge-distribution-3.0.0.Alpha1-offline.zip"
```

Comment:
It seems that Homebrew can’t parse the version from this URL; please add `version "3.0.0.Alpha1"` below this line, similar to the `version` line at the top of the formula:

```rb
devel do
  url "..."
  version "..."
  sha256 "..."
end
```